npc/re/other/dimensional_gap.txt

```diff
@@ -0,0 +1,549 @@
+		"I'm outta here~");
+	next();
+	mes("[Jumpy Knight]");
+	mes("Keep running you sneaky coward!");
+	hideonnpc("Merchant Prince#HUMERC2");
+	close2();
+	hideonnpc("Jumpy Knight#JP_RUN2");
+	hideoffnpc("Merchant Prince#HUMERC3");
+	disablenpc("Merchant Prince#HUMERC2");
+	end;
+
+OnInit:
+	disablenpc("Merchant Prince#HUMERC2");
+	end;
+OnTouch:
+	mes("[Prince Canet]");
+	mes("500 zeny");
+	close();
+}
+
+dali,67,113,3	script	Jumpy Knight#JP_RUN2	4_M_JP_RUN,{
+	end;
+
+OnInit:
+	hideonnpc("Jumpy Knight#JP_RUN2");
+	end;
+}
+
+dali,89,97,3	script	Merchant Prince#HUMERC3	4_M_HUMERCHANT,{
+	mes("[Prince Canet]");
+	mes("You are lucky that you have met up with me.\r"
+		"Who am I?\r"
+		"I'm here to become a merchant King.");
+	next();
+	mes("[Prince Canet]");
+	mes("I might be working a street stall now but I'll have an actual shop some day.");
+	next();
+	mes("[Prince Canet]");
+	mes("It's my first day so I'm having a major sale.");
+	next();
+	mes("[Prince Canet]");
+	mes("Let's see I've got a... Ring of something! Sword of stuff! A backpack of some sort!");
+	next();
+	mes("[Prince Canet]");
+	mes("Tell me if you need anything,");
+	hideoffnpc("Jumpy Knight#JP_RUN3");
+	next();
+	mes("[Prince Canet]");
+	mes("What are you doing here? This is my spot for a shop Get outta my turf!");
+	next();
+	mes("[Jumpy Knight]");
+	mes("Bah!!\r"
+		"You're gonna regret those words!");
+	next();
+	mes("[Prince Canet]");
+	mes("I'm shaking...");
+	hideonnpc("Merchant Prince#HUMERC3");
+	close2();
+	hideonnpc("Jumpy Knight#JP_RUN3");
+	enablenpc("Merchant Prince#HUMERC");
+	hideoffnpc("Merchant Prince#HUMERC");
+	end;
+
+OnInit:
+	hideonnpc("Merchant Prince#HUMERC3");
+	end;
+}
+
+dali,89,94,7	script	Jumpy Knight#JP_RUN3	4_M_JP_RUN,{
+	end;
+
+OnInit:
+	hideonnpc("Jumpy Knight#JP_RUN3");
+	end;
+}
+
+dali02,58,123,3	script	Curious Knight#KY_KNT	4_M_KY_KNT,{
+	mes("[Curious Knight]");
+	mes("The Royal Guard in here was desperate for help. We entered through a secret place in the palace.");
+	next();
+	mes("[Curious Knight]");
+	mes("We were whisked away here... No idea where I am but I have to establish a base here for the Knighthood.");
+	close();
+}
+
+dali02,51,112,5	script	Confused Thief#DST_SOL	4_DST_SOLDIER,{
+	mes("[Confused Thief]");
+	mes("One minute I'm in a bar... The next minute I'm here... What is this place?");
+	next();
+	mes("[Confused Thief]");
+	mes("There's no night or day in here. Hmm, something has to be valuable in this place...");
+	close();
+}
+
+dali02,65,115,5	script	Adventurous Rafflesia#ra	4_M_RAFLE_GR,{
+	mes("[Adventurous Rafflesia]");
+	mes("No no this has to be Thanatos Tower... I sense dark magic here.");
+	next();
+	mes("[Guide]");
+	mes("Oh, we're back here again. I cannot find a way out! We are trapped in this maze!");
+	next();
+	mes("[Tourist]");
+	mes("Is this place the famous Prontera? It is dark just like a big city!");
+	next();
+	mes("[Traveller]");
+	mes("I hear a ticking clock. This place must be Geffen!");
```

Comment:
I see all these npc show same 4-5 text, is it random? or they show this everytime in same order?